config/ccdb-migrate-job.yaml

```diff
@@ -29,7 +29,7 @@ spec:
             memory: 300Mi
           limits:
             cpu: 1000m
-            memory: 1Gi
+            memory: 4Gi
```

Comment:
I think it's unlikely that the database migrations use more than 4Gi -- my gut tells me it would probably be ~2Gi max on a large env -- but it would be awful if migrations got OOMKilled. Increased this to 4Gi since that is around what it would use on BOSH (kinda -- monit would have allowed it to temporarily burst higher for a few cycles).